rts/Sim/Misc/GlobalSynced.cpp

```diff
@@ -17,6 +18,8 @@
 	#include "System/Sync/SyncChecker.h"
 #endif
 
+const char* const tracingSpeedFactor = "SpeedFactor";
+const char* const tracingWantedSpeedFactor = "WantedSpeedFactor";
```

Comment:
Seems like these two should be `static constexpr const char*` ?

Reply:
It's not local to this translation unit, it's declared in the header and used in other place.